### PR TITLE
[rc] Angular 9

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: src/app/public/modules/angular-tree/angular-tree-node.component.html [hunk 1]
```diff
@@ -54,7 +50,11 @@
     <!-- https://github.com/500tech/angular-tree-component/blob/master/lib/components/tree-node-wrapper.component.ts -->
     <div
       class="node-content-wrapper"
+      role="treeitem"
       [allowDragoverStyling]="node.allowDragoverStyling()"
+      [attr.aria-current]="node.isActive"
+      [attr.aria-expanded]="node.isExpanded"
+      [attr.aria-selected]="ariaSelected()"
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/angular-tree/angular-tree-node.component.html:57]: ☝️ The `treeitem` role should be set on the element that is interactive and focusable. https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2a.html

File: package.json [hunk 2]
```diff
@@ -19,64 +20,60 @@
...
+    "@skyux/modals": "4.0.0-rc.2",
+    "@skyux/omnibar-interop": "4.0.0-rc.0",
+    "@skyux/popovers": "4.0.0-rc.1",
+    "@skyux/router": "4.0.0-rc.0",
+    "@skyux/tabs": "4.0.0-rc.0",
+    "@skyux/theme": "4.0.0-rc.2",
     "angular-tree-component": "8.5.6",
-    "codelyzer": "4.5.0",
+    "codelyzer": "5.2.2",
     "core-js": "2.6.11",
```
Comment (Blackbaud-SteveBrush) [inline package.json:72]: `core-js` and `@types/core-js` can be removed, since they're handled by Builder now.

File: src/app/app-extras.module.ts [hunk 3]
```diff
@@ -28,10 +33,20 @@ const treeModuleForRoot = TreeModule.forRoot();
...
   ],
   exports: [
+    SkyAngularTreeModule,
     SkyAppLinkModule,
+    SkyDocsToolsModule,
     SkyDropdownModule,
-    SkyAngularTreeModule,
     TreeModule
+  ],
+  providers: [
```
Comment (Blackbaud-SteveBrush) [inline src/app/app-extras.module.ts:42]: Are these docs changes not in the master branch already?

File: package.json (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply package.json:72]: Done.

File: src/app/app-extras.module.ts (hunk 3 above)
Comment (Blackbaud-AlexKingman) [reply src/app/app-extras.module.ts:42]: Nope. Should we release a 3 version with that before we release 4?

**Pull request merged**